test/tests/transfers/multiapp_high_order_variable_transfer/master_L2_Lagrange_userobject.i

```diff
@@ -0,0 +1,104 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 20
+  ny = 20
+  parallel_type = replicated
+[]
+
+[Variables]
+  [power_density]
+    family = L2_LAGRANGE
+    order = FIRST
+  []
+[]
+
+[AuxVariables]
+  [./multi_layered_average]
+    family = LAGRANGE
+    order = FIRST
+  [../]
+[]
+
+[UserObjects]
+  [./multi_layered_average]
+    type = LayeredAverage
+    variable = power_density
+    direction = y
+    num_layers = 4
+  [../]
+[]
+
+[AuxKernels]
+  [./layered_aux]
+    type = SpatialUserObjectAux
+    variable = multi_layered_average
+    execute_on = 'nonlinear TIMESTEP_END'
+    user_object = multi_layered_average
+  [../]
+[]
+
+[Functions]
+  [pwr_func]
+    type = ParsedFunction
+    value = '1e3*x*(1-x)+5e2'
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Reaction
+    variable = power_density
+  []
+
+  [coupledforce]
+    type = BodyForce
+    variable = power_density
+    function = pwr_func
+  []
+[]
+
+[Postprocessors]
+  [layered_avg]
+    type = ElementAverageValue
+    block = '0'
+    variable = multi_layered_average
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[Executioner]
+  type = Steady
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart '
+  petsc_options_value = 'hypre boomeramg 100'
+
+  nl_abs_tol = 1e-8
+  nl_rel_tol = 1e-12
+[]
+
+[MultiApps]
+  [sub]
```

Comment:
Oh - I just thought of a solution to your duplicate file problem. What you need to do is change the name of this MultiApp. That will automatically change the file pattern of the output files. Now it's not easy because we don't have a way to just rename the block with CLI override. Instead you'd have to supply each parameter's full path (e.g. `MultiApps/sub/type=FullSolveMultiApp MultiApps/sub/app_type=MooseTestApp MultiApps/sub/positions='0 0 0'`, etc.)

Reply:
You meant I should delete the block from the input files ? And then support them through args?

Reply:
I really want to delete all these duplicated  files.